packages/strapi-design-system/src/Combobox/Combobox.tsx

```diff
@@ -288,41 +285,20 @@ export const ComboboxInput = React.forwardRef<ComboboxInputElement, ComboboxInpu
  * Combobox
  * -----------------------------------------------------------------------------------------------*/
 
-interface ComboboxPropsWithoutLabel extends ComboboxInputProps, Pick<FieldProps, 'hint'> {
-  labelAction?: FieldLabelProps['action'];
-}
-
-export type ComboboxProps =
-  | (ComboboxPropsWithoutLabel & { label: string; 'aria-label'?: never })
-  | (ComboboxPropsWithoutLabel & { label?: never; 'aria-label': string });
+export interface ComboboxProps extends ComboboxInputProps {}
 
 export const Combobox = React.forwardRef<ComboboxInputElement, ComboboxProps>(
-  ({ error, hint, id, label, labelAction, required = false, ...restProps }, forwardedRef) => {
-    const generatedId = useId(id);
-
-    return (
-      <Field hint={hint} error={error} id={generatedId} required={required}>
-        <Flex direction="column" alignItems="stretch" gap={1}>
-          {label ? <FieldLabel action={labelAction}>{label}</FieldLabel> : null}
-          <ComboboxInput ref={forwardedRef} id={generatedId} error={error} required={required} {...restProps} />
-          <FieldHint />
-          <FieldError />
-        </Flex>
-      </Field>
-    );
+  ({ required = false, ...restProps }, forwardedRef) => {
+    return <ComboboxInput ref={forwardedRef} required={required} {...restProps} />;
```

Comment:
I don't think this component needs to be declared anymore? I think we could just export `ComboboxInput`?